pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```
@@ -116,24 +124,54 @@ spec:
     description: The buildah version used to build the model container image
     value: $(tasks.retrieve-build-image-info.results.buildah-version)
   tasks:
+  - name: fetch-model-registry
+    params:
+    - name: model-name
+      value: $(params.model-name)
+    - name: model-version
+      value: $(params.model-version)
+    - name: model-registry-hostname
+      value: $(params.model-registry-hostname)
+    - name: model-registry-enabled
+      value: $(params.model-registry-enabled)
+    - name: git-model-repo
+      value: $(params.git-model-repo)
+    - name: git-model-revision
+      value: $(params.git-model-revision)
+    - name: model-dir
+      value: model_dir-$(params.model-name)/
```

The source of the issue is this:
```suggestion
    - name: model-dir
      value: $(params.model-dir)
```

This directory is just referencing the directory for the Containerfile to copy the actual model files from. This directory is relative to the `CONTEXT` which for buildah task is: `model_dir-$(params.model-name)/$(tasks.fetch-model-registry.results.modelRelativePath)/$(params.model-name)`
So the 
```
checking on sources under "/workspace/source/model_dir-bike-rentals-auto-ml/bike-rentals-auto-ml": copier: stat: "/model_dir-bike-rentals-auto-ml": no such file or directory
```
actually meant I cannot find `/model_dir-bike-rentals-auto-ml` under `/workspace/source/model_dir-bike-rentals-auto-ml/bike-rentals-auto-ml`, which now makes complete sense but initially is confusing.